### PR TITLE
Invoke bintray API directly instead of with curl

### PR DESCRIPTION
**Pull request by tooda02**
This resolves issue [Use Go http package #2](https://github.com/drone-plugins/drone-bintray/issues/2) and updates to latest official Alpine image.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -3,8 +3,8 @@
 #     CGO_ENABLED=0 go build -a -tags netgo
 #     docker build --rm=true -t plugins/drone-bintray .
 
-FROM gliderlabs/alpine:3.1
-RUN apk-install ca-certificates
-RUN apk-install curl
+FROM alpine:3.3
+RUN apk update && apk add ca-certificates
 ADD drone-bintray /bin/
+COPY ca-certificates.crt /etc/ssl/certs/ca-certificates.crt
```
Comment (nlf) [inline Dockerfile]: is this needed? shouldn't installing ca-certificates be enough?

Comment (tooda02) [inline Dockerfile]: You'd think so.  But it didn't work when I tried it.

Comment (tboerger) [inline Dockerfile]: Than the ca certificate of alipne must be incomplete. Maybe report a bug there? I really dislike to maintain this file manually.

Comment (tboerger) [inline Dockerfile]: I would suggest to give alpine edge a try first

**Comment by tboerger** [comment]
Maybe we should just use a bintray client lib instead of building all of that manually ;)

File: Dockerfile (hunk 1 above)
Comment (tooda02) [inline Dockerfile]: No, this was my problem.  After making the previous comment, I remembered that I tested Alpine's certificate before adding the Go code to append it to the pool.  I retested, and this works with the Alpine cert.  I've removed the extra certificate from the merge.

**Comment by tooda02** [comment]
What's the status of the merge of this PR?

**Comment by tboerger** [comment]
LGTM

**Pull request merged**